File: 27. Remove Element/main.cpp
/*
https://leetcode.com/problems/remove-element/submissions/1240213781/
*/
class Solution {
public:
  int removeElement(vector<int> &nums, int val) {
    int cont = 0;
    for (int x = 0; x < nums.size(); x++)
      if (nums[x] != val) {
        nums[cont] = nums[x];
        cont++;
      }
    return cont;
  }
};

File: 27. Remove Element/Main.java
// https://leetcode.com/problems/remove-element/submissions/1240213781/
class Solution {
  public int removeElement(int[] nums, int val) {
    int nuevaLongitud = 0;
    for (int i = 0; i < nums.length; i++) {
      if (nums[i] != val) {
        nums[nuevaLongitud] = nums[i];
        nuevaLongitud++;
      }
    }

    return nuevaLongitud;
  }
}